File: herejong-back/herejongSupply/src/main/java/com/supply/herejongSupply/model/ProductStar.java
package com.supply.herejongSupply.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.index.Indexed;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "productStar")
public class ProductStar {

    @Id
	private String  id ;
	@Indexed
    private String productId ;
    private Integer starProductPoint ;
    

    public String getId() {
        return id;
    } 

    public void setId(String id) {
        this.id = id;
    }

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId;
    }

    public Integer getStarProductPoint() {
        return starProductPoint;
    }

    public void setStarProductPoint(Integer starProductPoint) {
        this.starProductPoint = starProductPoint;
    }

  
  

  



    
    
}

File: herejong-front/herejong-app/src/app/model/product-star.ts
export class ProductStar {
    id: string;
    productId: string;
    starProductPoint: number;
   

    constructor(
  
        productId: string,
        starProductPoint: number,
     

    ) {

        this.productId = productId;
        this.starProductPoint = starProductPoint;
      
     
    }
}
